bundles/models/models_service.go

```diff
@@ -85,6 +87,41 @@ func (ms *Service) GetModelProto(ctx context.Context, tx *gorm.DB, owner,
 func (ms *Service) ModelList(p *gz.PaginationRequest, tx *gorm.DB, owner *string,
 	order, search string, likedBy *users.User, user *users.User, categories *category.Categories) (*fuel.Models, *gz.PaginationResult, *gz.ErrMsg) {
 
+	// Get a boolean that indicates if this a basic `GET /models` or `GET /models?page=N` query.
+	// In this case, we can ideally use the memdory cache to reduce the
+	// DB burden.
+	// Note: the PerPage default value is 20.
+	basicQuery := owner == nil && order == "" && search == "" && likedBy == nil && p != nil && (!p.PageRequested || (p.PageRequested && p.PerPage == 20)) && tx.Value == nil
```

Comment:
This could be a function

Reply:
Done in [ca689c0](https://github.com/gazebo-web/fuel-server/pull/21/commits/ca689c01570d686f24b99050b213273719ec92fc)